CHANGELOG.md

```diff
@@ -14,7 +14,14 @@
 
 ## Compatibility Notes
 
--
+- [eslint-config] Switch the following `jsdoc` related levels from `warn` to `error`: 
```

Comment:
You need to create a `changeset` using the changeset command. Then with space/enter select the packes which have changed and what should be bumped. Please check in the generated file.